File: app/src/main/java/com/dev/touyou/ffmultiplier/Model/DatabaseScore.java
package com.dev.touyou.ffmultiplier.Model;

import com.google.firebase.database.IgnoreExtraProperties;

/**
 * Created by touyou on 2016/12/10.
 */
@IgnoreExtraProperties
public class DatabaseScore {
    public String name;
    public int score;

    public DatabaseScore() {}

    public DatabaseScore(String name, int score) {
        this.name = name;
        this.score = score;
    }

    public int getScore() {
        return score;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/kotlin/com/dev/touyou/ffmultiplier/Model/DatabaseScore.kt
package com.dev.touyou.ffmultiplier.Model

import com.google.firebase.database.IgnoreExtraProperties

/**
 * Created by touyou on 2016/12/10.
 */
@IgnoreExtraProperties
class DatabaseScore {
    var name: String
    var score: Int = 0

    constructor() {}

    constructor(name: String, score: Int) {
        this.name = name
        this.score = score
    }
}
